jsonview/decorators.py

```diff
@@ -30,9 +30,23 @@
 
 
 def _dump_json(data):
+    options = getattr(settings, 'JSON_OPTIONS', {})
+
     if getattr(settings, 'JSON_USE_DJANGO_SERIALIZER', True):
```

Comment:
should prioritize `options` over `JSON_USE_DJANGO_SERIALIZER` and add a `DeprecationWarning` to using this arg—`JSON_OPTIONS` is strictly more powerful and is The Future™